001-intro.md

```diff
@@ -100,96 +100,358 @@ to real life cases involving actual DMs.**
 Use Cases {#sec:usecases}
 =========
 
+General Remarks
+---------------
+
 This specification provides a standardized mechanism
-for annotating VOTables. In this sense, it enables an indefinite number
-of use cases, each specific to the Data Model instances being represented
-in VOTable format.
-
-However, to give a sense of what it is possible to accomplish with this
-specification, we provide some explicit use cases relative to the VO
-domain.
-
-A typical usage scenario may be a VOTable naïve (see [@sec:clients]) client
-that is sensitive to certain models only, say STC. Such a tool can be
-written to understand annotation with STC types. Finding an element
-mapped to a type definition from STC it might infer for example that it
+for annotating VOTables according to data models. Thus, it enables:
+
+  * Data providers to annotate VOTables so to faithfully map VOTable contents to one or more
+  data models, as long as such Data Models are expressed according to the
+  VO-DML standard [@vodml]. In other terms, they can *serialize* data model
+  *instances* in a standard, interoperable way. Some examples are provided below
+  as concrete use cases.
+  * Service clients to faithfully reconstruct the semantics of the data model instances
+  they consume in VOTable format. Some concrete examples are also provided below.
+
+One of the main goals of this specification is also to alleviate data modelers

```

Comment:
@lmichel I guess you are referring to these lines when you say:

> Paragraph #2 et #3 in section 2.1 should rather talk about mapping instead of model serialization.

This is referring to the old pattern where documents would define data models and their serialization, e.g. in SpectrumDM they define the data model and then they show the VOTable and FITS files that are compatible with that data model. There is no separation between the data model, instances of types defined therein, and serializations: it's all in the same document.

So I am not sure I understand your comment and how to improve the current text.

By the way, it's way easier to follow the conversation if one comments the specific diff part (as I am doing right now) rather than submitting individual code review summaries for each specific comment referring to a specific change.